pkg/engine/context/loaders/globalcontext.go

```diff
@@ -106,48 +106,20 @@ func (g *gctxLoader) loadGctxData() ([]byte, error) {
 		return nil, err
 	}
 
-	var jsonData []byte
-	if _, ok := data.([]byte); ok {
-		jsonData = data.([]byte)
-	} else {
-		jsonData, err = json.Marshal(data)
+	if rc.JMESPath == "" {
+		jsonData, err := json.Marshal(data)
 		if err != nil {
 			return nil, err
 		}
-	}
-	g.logger.V(6).Info("fetched json data", "name", g.entry.Name, "jsondata", jsonData)
-
-	if g.entry.GlobalReference.JMESPath == "" {
-		err := g.enginectx.AddContextEntry(g.entry.Name, jsonData)
```

Comment:
We no longer need to add it to the context variables?

Reply:
This is duplicate code, context will be added https://github.com/kyverno/kyverno/blob/v1.13.4/pkg/engine/context/loaders/globalcontext.go#L67